### ISSUE TITLE
Fundraising Ideas Tool

### ISSUE THREAD

**Issue by LaurenMcM749** [Amaxa-Impact/amaxa#10]
Problem: It is difficult for students to come up with fundraising ideas.

Solution: Coaching is a large part of this solution. However, we can provide a database that allow students to filter ideas based on their interest, location, and more. 

Priority: Medium

**Comment by not-ani** [comment]
Wouldn't this just be action guides?

**Comment by LaurenMcM749** [comment]
No, basically just simple table that has a huge list of like 200+ ideas filtered by tagged interest. But it's just the idea, not any info/resources on how to do it. Like for example "Yard Sale" or "Sell stickers". Eventually though we could use this data to figure out the kinds of ideas people are clicking on/not and why, how many they look through before they choose one, if they choose frmo list, etc

**Comment by not-ani** [comment]
would this be per project or for everyone?
